operations/operations/zuul-ci-cd-quickstart-user-guide.md

```diff
@@ -0,0 +1,236 @@
+
+We consider that zuul knows about your repository so we can get started. There are three
+topics that you should know about. To get jobs running you need the "job" itself. Jobs run
+within "pipeline". The third important thing is to provide a "project" definition.
+
+## Where to save the zuul relavant data?
+
+Zuul will parse all branches of the untrusted repositories that zuul knows about.
+Your repository is most likley an untrusted one since only the configuration repostiories should
+have the "trusted" state.
+So it doesn't matter whether you have just one branch containing zuul files or all branches. Zuul
+is looking for the following pathes on your repositories root.
+
+```bash
+./zuul.yaml # everything is in here
+
+./.zuul.yaml # ... or here
+
+./zuul.d/ # use directory style to get a bit of a structure
+├── jobs.yaml
+└── project.yaml
+
+./.zuul.d/ # the same as before just hidden
+├── jobs.yaml
+└── project.yaml
+```
+
+Just use exact on of the four possiblities.
+
+If using the directory style configuration all `yaml` files within in this directory will be
+processed. If your projects configuration is small enough you may put all information in
+a single file called `zuul.yaml`. It is also possible to create the file or the directory
+with a leading dot to hide them for non zuul related work within the repository.
+
+### Projects
+
+If zuul is configured to observe your repository it will have a look at your projects
+definition. Minimal example:
+
+```yaml
+- project:
+    name: my-org/my-repo
+    default-branch: main
+    merge-mode: "squash-merge"
+    my_pipeline1:
+      jobs:
+        - my_job1
+        - my_job2
+        ......
+    my_pipeline2:
+      jobs:
+        - my_jobs
+    ...
+
+```
+
+By default zuul will observe all branches for such files. We have to set the repository name
+that have to match the exact value that was set for zuul. Set a default-branch where actions
+that don't match an explicit branch are executed on. Set the merge-mode that zuul has to use.
+But beware that not all issue tracker support all methods. For github squash-merge will work.
+
+After these three properties add the pipelines you want to use to the project definition.
+With the `jobs` list you define which jobs to run in which pipeline.
+
+### Pipelines
+
+Every zuul instance will have at least one repository that is used for configuration. There
+you will find the available pipelines. Pipelines are used to run your jobs on a periodic or
+event driven base. Pipelines can be used to run other pipelines and to keep your jobs in a
+defined order if you need this.
+
+Have a look at the configuration repository to utilize the pipelines for your repository. You
+are not able to define new pipelines outside of a so called "configuration" repository. Since,
+by default your repo is considered "untrusted". So in the first place you don't need to
+think about, how to create a pipeline. Just use one that fits your needs as close as possible.
```

Comment:
If one doesn't have to think about it, maybe it shouldn't be mentioned in the quickstart guide at all.

Instead, you could add a list of pipelines that are configured and when to use them. Like "check" for tests to run on a PR or "periodic" for cron-like regular testing.

Reply:
i would leave that as is. But i added an enumeration of the available pipelines and in which cases they
are triggered.